### PR TITLE
[UI] Responsive Pages

### LINKED ISSUE #567: [UI] meshery page needs to be more responsive
**Description**
As we  squish meshery page to 50% of the screen it's not responsive but if we squish it further then its responsive.
Screenshot from 2020-01-06 08-55-42
Screenshot from 2020-01-06 09-05-19

### PR DESCRIPTION
**Pull request by dhruv0000**
**Description**

This PR fixes #567  

**Notes to reviewers**

- Changed the breakpoints for the Dashboard, Performance test and Meshery Play components.
- Fixed provider Meshery logo for small screens.

#### Checklist

- [x] [Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).
- [x] PR title includes `[<name of area>]:<description>`

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
Is the attached image taken after fixing yhe issue for provider page?
@dhruv0000 ?

**Comment by dhruv0000** [comment]
> Is the attached image taken after fixing the issue for provider page?
> @dhruv0000 ?

Before @Nikhil-Ladha

**Comment by Nikhil-Ladha** [comment]
> > Is the attached image taken after fixing the issue for provider page?
> > @dhruv0000 ?
> 
> Before @Nikhil-Ladha .
> Here is the new one.
> image

Ohh cool. Seemed like that to me too, but just wanted to confirm😅 as it was attached with the PR. Nice!

**Comment by kushthedude** [comment]
@dhruv0000 Nice work here! But I guess you would want to reiterate this PR

Screenshot 2020-07-10 at 9 07 29 PM

Screenshot 2020-07-10 at 9 08 52 PM

Breakpoints need to be changed for each route where we have input fields.

**Comment by kushthedude** [comment]
@dhruv0000 Pardon me for changing the widths of your uploaded image, it was too frustrating to scroll all down 😄

**Comment by dhruv0000** [comment]
> @dhruv0000 Nice work here! But I guess you would want to reiterate this PR
> 
> Screenshot 2020-07-10 at 9 07 29 PM
> 
> Screenshot 2020-07-10 at 9 08 52 PM
> 
> Breakpoints need to be changed for each route where we have input fields.

Cool, Working on the Settings Panel.
The Performance Test shouldn't suppose to happen. Let me check it again.

**Comment by dhruv0000** [comment]
@kushthedude @Nikhil-Ladha . Take a look now. Most of the bugs are fixed, except this one, due to the text.
One of the workaround would be to remove the labels for smaller screens.
Would be a big UI change (if someone uses meshery in 700px width screen :thinking: ), so wanted to run by you first.

**Comment by kushthedude** [comment]
Is it just an edge case at 700px?

**Comment by dhruv0000** [comment]
700px and below.
Chcek it out yourself:
https://r2qd0.sse.codesandbox.io/settings#metrics/grafana (temp link)

**Comment by Nikhil-Ladha** [comment]
@dhruv0000 you can do one thing is that give the label field breakpoint, after that add the ellipsis property to it make the lavel look something like 'Environ..' . How does that sound?

As generally we won't get such a user having width below 700 px, but if the above one doesn't works we can add a min-width property for the whole div of settings so that the labels don't break or have a horizontal scroll.

**Comment by Nikhil-Ladha** [comment]
Also, you may want to check this as well.
Screenshot_20200715-181009__01

**Comment by kushthedude** [comment]
> Also, you may want to check this as well.
> Screenshot_20200715-181009__01

We wont have any mobile users, as well as users with screen size<700px since meshery is not a traditional web application. I guess the main issue here is solved and we can avoid the mobile responsiveness as no one would use meshery on a mobile.

**Comment by leecalcote** [comment]
@dhruv0000 demo on today's Dev call? 

I'll explain why eventually we will have mobile users and that we do want to support mobile interfaces. There's forward progress in this PR that we can accept and a new issue that can be created to finish the job

**Comment by kushthedude** [comment]
Closed this to have a look at netlify checks.

File: ui/components/MesherySettingsPerformanceComponent.js [hunk 1]
```diff
@@ -25,6 +25,10 @@ const styles = (theme) => ({
   root: {
     padding: theme.spacing(10),
   },
+  textField: {
+    // minWidth: '190px',
+    // minMargin: 0,
+  },
```
Comment (kushthedude) [inline ui/components/MesherySettingsPerformanceComponent.js:31]: @dhruv0000 This is useless I guess? Any reason to have this?

Comment (dhruv0000) [reply ui/components/MesherySettingsPerformanceComponent.js:31]: Nope. Just kept for any future changes.
Would confuse future contributions :thinking:.
Let me remove it.

**Pull request merged**